### PR TITLE
Rem 614

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
fix physical in item form

### CODE REVIEW

File: src/pages/iv-items/forms/PhysicalForm.js [hunk 1]
```diff
@@ -152,7 +149,7 @@ const PhysicalForm = ({ labels, editMode, maxAccess, store }) => {
       }))
     } catch (error) {}
   }
-
+  console.log(formik.values, 'values')
```
Comment (imtithalhalalo) [inline src/pages/iv-items/forms/PhysicalForm.js:152]: remove console.log

**Comment by omarZrayka23** [comment]
I didn't see the get request when I set the value to zero.

**Comment by omarZrayka23** [comment]
Screenshot 2024-10-21 at 1 04 12 PM

title ?!!

File: src/pages/iv-items/forms/PhysicalForm.js [hunk 2]
```diff
@@ -229,8 +238,8 @@ const PhysicalForm = ({ labels, editMode, maxAccess, store }) => {
                 label={labels.length}
                 value={formik.values.length}
                 maxAccess={maxAccess}
-                onChange={e => handleFieldChange('length', 1, e.target.value)}
-                onClear={() => handleFieldChange('length', 1, 0)}
+                onMouseLeave={e => handleFieldChange('length', 1, e?.target?.value)}
+                onClear={() => handleFieldClear('length', 1, 0)}
```
Comment (omarZrayka23) [inline src/pages/iv-items/forms/PhysicalForm.js:242]: remove last param zero

File: src/pages/iv-items/forms/PhysicalForm.js [hunk 3]
```diff
@@ -219,8 +228,8 @@ const PhysicalForm = ({ labels, editMode, maxAccess, store }) => {
                 value={formik.values.diameter}
                 maxAccess={maxAccess}
                 readOnly={formik.values?.shape === 1}
-                onChange={e => handleFieldChange('diameter', 4, e.target.value)}
-                onClear={() => handleFieldChange('diameter', 4, 0)}
+                onMouseLeave={e => handleFieldChange('diameter', 4, e?.target?.value)}
```
Comment (omarZrayka23) [inline src/pages/iv-items/forms/PhysicalForm.js:231]: you can simplify your code by passing e (event) directly and then assigning e?.target?.value to value inside the handleFieldChange function. ex:  const handleFieldChange = (fieldName, e) => {
  const value = e?.target?.value; }

**Pull request merged**